packages/sdk-browser/README.md

```diff
@@ -12,27 +12,126 @@ Turnkey API documentation lives here: https://docs.turnkey.com.
 $ npm install @turnkey/sdk-browser
 ```
 
+### Initialize
+
+```typescript
+import { Turnkey } from "@turnkey/sdk-browser";
+
+const turnkey = new Turnkey({
+  apiBaseUrl: "https://api.turnkey.com",
+  defaultOrganizationId: process.env.TURNKEY_ORGANIZATION_ID,
+  // Optional: Your relying party ID - for use with Passkey authentication
+  rpId: process.env.TURNKEY_RP_ID,
+});
+```
+
+### Turnkey Clients
+
+#### Passkey
+
+The Passkey client allows for authentication to Turnkey's API using Passkeys.
+
+```typescript
+const passkeyClient = turnkey.passkeyClient();
+
+// User will be prompted to login with their passkey
+await passkeyClient.login();
+
+// Make authenticated requests to Turnkey API, such as listing user's wallets
+const walletsResponse = await passkeyClient.getWallets();
+```
+
+#### Iframe
+
+The Iframe client can be initialized to interact with Turnkey's hosted iframes for sensitive operations.
+The `iframeContainer` parameter is required, and should be a reference to the DOM element that will host the iframe.
+The `iframeUrl` is the URL of the iframe you wish to interact with.
+
+The example below demonstrates how to initialize the Iframe client for use with [Email Auth](https://docs.turnkey.com/embedded-wallets/sub-organization-auth)
+by passing in `https://auth.turnkey.com` as the `iframeUrl`.
+
+```typescript
+const iframeClient = await turnkey.iframeClient({
+  // The container element that will host the iframe
+  iframeContainer: document.getElementById("<iframe container id>"),
+  iframeUrl: "https://auth.turnkey.com",
+});
+
+const response = await iframeClient.injectCredentialBundle(
```

Comment:
tiny nit: let's use something like `injected` here given I don't think the response for `injectCredentialBundle` is meaningful

Reply:
@andrewkmin are you suggesting just renaming the variable?